howso/residuals.amlg

```diff
@@ -502,6 +502,18 @@
 							)
 						)
 
+						;if holding out, filter out all case ids provided
+						(declare (assoc
+							filtering_list
+							(if holdout case_ids (list case_id) )
```

Comment:
This line should be indented, and the else condition should probably give an empty `(list)`

Reply:
the else condition is the (list case_id), i'll format to look better